src/main/groovy/wooga/gradle/snyk/tasks/Report.groovy

```diff
@@ -0,0 +1,27 @@
+package wooga.gradle.snyk.tasks
+
+
+import wooga.gradle.snyk.cli.options.ProjectOption
+import wooga.gradle.snyk.cli.options.TestOption
+
+import javax.inject.Inject
+
+class Report extends Test {
+
+    @Inject
+    Report() {
+        reports.json.enabled = true
+    }
+
+    @Override
+    protected Boolean getIgnoreExitValue() {
+        true
+    }
+
+    @Override
+    void addMainOptions(List<String> args) {
```

Comment:
Does this need to be overridden if it's the same implementation as the in its base, Test?

Reply:
To be fair I didn't get this method at all. I would have implemented that with a `get` style method. `Report` is a direct sub-class. So from my perspective it works

Reply:
The method is for adding the internal arguments used by the task. Since `Report` derives from `Test` and has the same arguments, there's no need to override.